tests/expected/assert-location/assert-false/expected

```diff
@@ -1,3 +1,3 @@
-line 17 assertion failed: false: FAILURE
```

Comment:
AFAIK, this test is intentionally checking for line number since there was a related bug that @celinval fixed.